brim/release

```diff
@@ -150,8 +150,8 @@ install_zeek_package salesforce/ja3 133f2a128b873f9c40e4e65c2b9dc372a801cf24
 #
 
 mkdir -p zeek/bin zeek/lib/zeek zeek/share/zeek
-cp brim/zeekrunner$exe zeek/
-cp /usr/local/zeek/bin/zeek$exe zeek/bin/
+cp brim/zeek$exe zeek/bin
+cp /usr/local/zeek/bin/zeek$exe zeek/bin/zeekcmd$exe
```

Comment:
Renaming `zeek/bin/zeek$exe` to `zeek/bin/zeekcmd$exe` feels odd. How about leaving it as is and just renaming `zeek/zeekrunner$exe` to `zeek/zeek$exe` (which would continue to invokes `$dir/bin/zeek`)?

Reply:
I don't like this because I would like to have this work like the regular zeek build package, meaning the zeek binary should be in bin.